src/frontend/screens/Game/GameSubMenu/index.tsx

```diff
@@ -410,6 +410,14 @@ export default function GamesSubmenu({
               {t('button.browse_wine_prefix', 'Browse Wine Prefix')}
             </button>
           )}
+          {isInstalled && (
+            <button
+              onClick={() => window.api.showGameConfigFileInFolder(appName)}
```

Comment:
to answer your question, I think you can use `window.api.showConfigFileInFolder(appName)` here?